### ISSUE TITLE
Routes of attached engines not showing

### ISSUE THREAD

**Issue by skoona** [schneems/sextant#37]
My Rails 3.2 application has three Rails Engines mounted with over 50+ routes.  Sextant is only showing the 15 or so routes that belong to the parent application, none for the mounted engines.

Is there a way for the mounted_routes to be included?

Thanks, 
(Great Gem BTW)

**Comment by schneems** [comment]
Do they show when you `$ rake routes`? I would recommend upgrading rails versions, 3.2 is really old and no longer supported by Rails core. If you don't I would recommend Rails LTS at the very least https://railslts.com/. Sextant is included in Rails 4

**Comment by skoona** [comment]
Yes, rake routes shows them all by engine.  

using rails 3.2.22 with jruby-1.7.17 in this app.  I have another app based on Rails 4.2.5 and was using /rails/info/routes there; loved it, which is why i went looking for this gem.   Rails upgrades are a really big deal for established apps, but we are working on it.

Not having all the routes is not the end of the world, but if it we moderately possible; it would be helpful.

**Comment by schneems** [comment]
We are getting that data from Rails. If Rails isn't exposing it, there's not much we can do to fix the problem https://github.com/schneems/sextant/blob/f111fec8a3b89c2f19fc0c859e834ea7d47c6164/lib/sextant.rb#L6-L12

**Comment by skoona** [comment]
The data is there, because 'rake routes' returns it.  It cd Rails.application.routes.named_routes includes a list of mounted engines, and some entries have a reference to the app object for that engine.  Asking each engine for their routes, does get to the data.

I personally haven't worked out the details, hoping you are more familiar.

**Comment by schneems** [comment]
The last time I touched this code was almost 3 years ago. I wasn't planning on doing further dev on it since 3.2 is way past EOL and feature was merged into Rails 4. If you have a fix, I could merge and cut a new release.

**Comment by skoona** [comment]
Ok, I will look into it later.  I don't have a fix, but I'm sure an expansion is possible.  Just need to determine which route entries are actually engines, and ask that engine (only once) for its routes.

Rails.application.routes.routes and Rails.application.routes.mounted_routes appear to contain the app object that would give us routes for that engine.  Just need to sort out the navigation.

something like this might work:

```ruby

 l_routes = Rails.application.routes.routes 
Rails.application.routes.named_routes.each {|e| l_routes << e.app.routes.routes if e.app.present? }
l_routes

```

Admittedly I haven't tried it, but I will look into it and let you know.

**Comment by skoona** [comment]
After a quick look into the rails code, I see this will take a little more effort.  I will spend more time on it later, and let you know.

**Comment by skoona** [comment]
ok, so I've looked at the delta between 3.2 and 4.25 codesets related to /rails/info/routes.  The delta is too large to warrant a fixup for 3.2.   Interesting stuff, and in my idle time I may come back to it.  But its true;y not needed for what I/we are doing today.

I take it back, I don't want this feature in 3.2
